### DISCUSSION TITLE
Use CameraControls with GizmoHelper?

### DISCUSSION THREAD

**Discussion by RoboticWater** [pmndrs/drei#1359] [Q&A]
Is it possible to use the `GizmoHelper` widget with a `CameraControls` camera? I can get it to render and rotate correctly in [this CodeSandbox example](https://codesandbox.io/s/react-three-fiber-camera-controls-forked-9egcy7?file=/src/App.tsx), but I get a `Cannot use 'in' operator to search for 'minPolarAngle' in null` error once I click on an axis and the camera rotation gets messed up.

I've tried setting the `onTarget` and `onUpdate` methods, but the former doesn't seem to change anything, and the latter doesn't seem to work with the `update` method on the `CameraControls`, specifically `CameraControls.update()` expects a delta parameter which isn't supplied in `GizmoHelper`'s `onUpdate` (presumably because `OrbitControls`'s `update` method doesn't need a delta parameter.

**Comment by saumac** [comment]
Setting "makeDefault" prop on your controls should fix it:
Link: https://github.com/pmndrs/drei#gizmohelper:~:text=Make%20sure%20to%20set%20the%20makeDefault%20prop%20on%20your%20controls%2C%20in%20that%20case%20you%20do%20not%20have%20to%20define%20the%20onTarget%20and%20onUpdate%20props.
